ee/tabby-db/migrations/0022_github-provider.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE github_provider (
```

Comment:
```suggestion
CREATE TABLE github_repository_provider (
```

Update other naming correspondingly